spec/models/lesson_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Lesson, type: :model do
+  context 'when validating presence' do
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_presence_of(:teacher_subject_id) }
+    it { is_expected.to validate_presence_of(:rc_course_id) }
+    it { is_expected.to validate_presence_of(:price) }
```

Comment:
No need to validate the foreign key. You already have the test for the associations